frontend/src/pages/Forum.tsx

```diff
@@ -7,25 +7,41 @@ import Header from '../components/Header';
 import { Collapse } from '@geist-ui/core';
 import { useNavigate } from 'react-router-dom';
 
-interface Message {
-    id: number,
-    user: number,
-    content: string,
-}
+// interface Message {
+//     id: number,
+//     user: number,
+//     content: string,
+// }
+
+// interface Post {
+//     id: number,
+//     user: number,
+//     status: string,
+//     date_posted: Date,
+//     title: string,
+//     content: string,
+//     messages: number[]
+// }
 
 interface Post {
-    id: number,
-    user: number,
-    status: string,
-    date_posted: Date,
-    title: string,
-    content: string,
-    messages: number[]
+  authorId: string
+  datePosted: Date
+  content: string
+}
+
+interface ForumThread {
+  id: string,
+  authorId: string,
+  datePosted: Date,
+  content: string,
+  title: string,
+  status: string,
+  posts: Post[]
 }
 
 function Forum() {
-  const [postList, setPostList] = useState<Array<Post>>([]);
-  const [messagesList, setMessagesList] = useState<Array<Message>>([]);
+  const [postList, setPostList] = useState<Array<ForumThread>>([]);
+  const [messagesList, setMessagesList] = useState<Array<ForumPost>>([]);
```

Comment:
This variable isn't used and I don't think it makes sense to have here, since I imagine that one Array corresponds to a single ForumThread